my-app/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true
+}
```

Comment:
what is this?
why do we have to create this file?

Reply:
By mistake it happens I have checked the app it work fine without it.